QuickLookCard/scripts/mods/QuickLookCard/QuickLookCard_localization.lua

```diff
@@ -249,110 +249,137 @@ return {
 		en = "BMBR", -- Damage Resistance (Bombers)
 		["zh-cn"] = "雷兵", -- 伤害抗性（轰炸者）
+		ja = "手榴", -- ダメージ耐性 (ボマー)
 	},
 	trait_weapon_trait_melee_common_wield_increased_resistant_damage = {
 		en = "UYLD", -- Damage (Unyielding Enemies)
 		["zh-cn"] = "不屈", -- 伤害（不屈敌人）
+		ja = "不屈", -- ダメージ (不屈の敵)
 	},
 	trait_weapon_trait_melee_common_wield_increased_disgustingly_resilient_damage = {
 		en = "IFST", -- Damage (Infested Enemies)
 		["zh-cn"] = "感染", -- 伤害（感染敌人）
+		ja = "感染", -- ダメージ (感染した敵)
 	},
 	trait_weapon_trait_melee_common_wield_increased_unarmored_damage = {
 		en = "UAMR", -- Damage (Unarmoured Enemies)
 		["zh-cn"] = "无甲", -- 伤害（无甲敌人）
+		ja = "未装", -- ダメージ (アーマー未装着の敵)
 	},
 	trait_weapon_trait_melee_common_wield_increased_berserker_damage = {
 		en = "MNAC", -- Damage (Maniacs)
 		["zh-cn"] = "狂人", -- 伤害（狂人）
+		ja = "狂信", -- ダメージ (マニアック)
 	},
 	trait_weapon_trait_melee_common_wield_increased_super_armor_damage = {
 		en = "CRPC", -- Damage (Carapace Armoured Enemies)
 		["zh-cn"] = "硬壳", -- 伤害（硬壳装甲敌人）
+		ja = "重装", -- ダメージ (カラペース・アーマー装着の敵)
 	},
 	trait_weapon_trait_melee_common_wield_increased_armored_damage = {
 		en = "FLAK", -- Damage (Flak Armoured Enemies)
 		["zh-cn"] = "防弹", -- 伤害（防弹装甲敌人）
+		ja = "軽装", -- ダメージ (フラック・アーマー装着の敵)
 	},
 	trait_weapon_trait_ranged_common_wield_increased_resistant_damage = {
 		en = "UYLD", -- Damage (Unyielding Enemies)
 		["zh-cn"] = "不屈", -- 伤害（不屈敌人）
+		ja = "不屈", -- ダメージ (不屈の敵)
 	},
 	trait_weapon_trait_ranged_common_wield_increased_disgustingly_resilient_damage = {
 		en = "IFST", -- Damage (Infested Enemies)
 		["zh-cn"] = "感染", -- 伤害（感染敌人）
+		ja = "感染", -- ダメージ (感染した敵)
 	},
 	trait_weapon_trait_ranged_common_wield_increased_unarmored_damage = {
 		en = "UAMR", -- Damage (Unarmoured Enemies)
 		["zh-cn"] = "无甲", -- 伤害（无甲敌人）
+		ja = "未装", -- ダメージ (アーマー未装着の敵)
 	},
 	trait_weapon_trait_ranged_common_wield_increased_berserker_damage = {
 		en = "MNAC", -- Damage (Maniacs)
 		["zh-cn"] = "狂人", -- 伤害（狂人）
+		ja = "狂信", -- ダメージ (マニアック)
 	},
 	trait_weapon_trait_ranged_common_wield_increased_super_armor_damage = {
 		en = "CRPC", -- Damage (Carapace Armoured Enemies)
 		["zh-cn"] = "硬壳", -- 伤害（硬壳装甲敌人）
+		ja = "重装", -- ダメージ (カラペース・アーマー装着の敵)
 	},
 	trait_weapon_trait_ranged_common_wield_increased_armored_damage = {
 		en = "FLAK", -- Damage (Flak Armoured Enemies)
 		["zh-cn"] = "防弹", -- 伤害（防弹装甲敌人）
+		ja = "軽装", -- ダメージ (フラック・アーマー装着の敵)
 	},
 	trait_weapon_trait_increase_stamina = {
 		en = "STAM", -- Stamina
 		["zh-cn"] = "体力", -- 体力
+		ja = "活力", -- スタミナ
 	},
 	trait_gadget_stamina_regeneration = {
 		en = "STRG", -- Stamina Regeneration
 		["zh-cn"] = "体回", -- 体力恢复
+		ja = "活回", -- スタミナ回復
 	},
 	trait_gadget_cooldown_reduction = {
 		en = "ABRG", -- Combat Ability Regeneration
 		["zh-cn"] = "技回", -- 作战技能恢复
+		ja = "能回", -- 戦闘アビリティ回復
 	},
 	trait_weapon_trait_increase_impact = {
 		en = "IMPC", -- Impact (Melee)
 		["zh-cn"] = "冲击", -- 冲击（近战）
+		ja = "衝撃" -- 衝撃
```

Comment:
Please add the trailing comma here, so next guy wont need to remember that

Reply:
Added.